### DISCUSSION TITLE
Thumbnails plugin failing

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#814] [General]
Hi,

Just installed 0.55.0 alpha1. I get a messge: Plugin is failing. It seems to be the Thumbnails plugin. See screenshot. In the second screenshot you can see I tried to start it again.

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

![image](https://github.com/user-attachments/assets/e38e9b77-eef9-4358-a2b1-266bf505d017)

![image](https://github.com/user-attachments/assets/a79db9ac-1eb0-4f49-be5b-81ac0341b1d0)

**Comment by rejetto** [comment]
check sharp plugin

**Reply by PH5HP** [reply]
Hi,
TNX. It seems to work OK now. Did I miss something, because I did not had the Sharp plugin installed before (I think). It was not installed .

Kind regards,
Henk - ph5hp.nl - fileserver.ph5hp.nl

**Comment by rejetto** [comment]
it was surely installed, as it's a requirement of "thumbnails"

**Comment by rejetto** [comment]
sharp plugin is just a (big) part of thumbnails that i decided to keep separated so that you download sharp only once, and i can provide you updates for "thumbnails" that are very small
